app/middlewares/auth_middleware.py

```
@@ -0,0 +1,42 @@
+    user = None
+    for u in USERS.values():
+        if u.id == int(user_id):
+            user = u
+            break
+    
+    if user is None:
+        raise credentials_exception
+        
+    return user
+
+async def get_current_active_user(current_user = Depends(get_current_user)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)